app/helpers/signs_helper.rb

```diff
@@ -17,4 +17,12 @@ def sign_status_orb(sign)
     orb_class = sign.private? ? 'sign-private' : sign.active? ? 'sign-active' : 'sign-inactive'
```

Comment:
I put it in there initially but changed it to `signs_helper.rb` because [this line](https://github.com/chapmanu/signage/blob/development/app/views/signs/play.html.erb#L17) in the page that David wanted the exception for muted videos in led me to [here](https://github.com/chapmanu/signage/blob/development/app/models/sign.rb#L37).

So it led me to believe I should be doing this in `signs_helper.rb`. Should it still be in `slides_helper.rb` instead?